animator/src/animations/rainbow_sphere.rs

```diff
@@ -1,18 +1,47 @@
 use super::Animation;
 use rustmas_light_client as client;
+use serde::Deserialize;
+use serde_json::json;
+
+#[derive(Deserialize)]
+struct Parameters {
+    x: f64,
+    y: f64,
+    z: f64,
```

Comment:
nit: naming could be more verbose - it could be immediately clear that these are coordinates of the center offset